### DISCUSSION TITLE
Sprint 4 meeting | 2022-11-03

### DISCUSSION THREAD

**Discussion by HuangJunye** [QPong/planning#34] [Meetings]
## Agenda

- [x] Checkpoint 2: https://github.com/qiskit-advocate/qamp-fall-22/issues/26#issuecomment-1302034632
- [x] Sprint review (10 min)
- [x] Sprint retro (10 min)
- [x] Sprint planning (10 min)
- [x] Q&A (5 min)
- [x] Game dev resources and news (5 min)
  - [x] God of War Ragnarök is releasing on next week (Nov 9)
  - [x] God of War - Raising Kratos: Full Length Feature Documentary https://www.youtube.com/watch?v=lJZWKBDXXFY
  - [x] Reinventing God of War | GDC 2019: https://www.youtube.com/watch?v=aIb-Lt7WX_s

**Comment by HuangJunye** [comment]
## Review

[Last sprint planning](https://github.com/QPong/planning/discussions/32#discussioncomment-3904465)

### Dayeong

### Unathi

### Siddharth

### Junye 

### Follow up

**Reply by tula3and** [reply]
I had two hackathons with Unathi and got used to using PICO-8 with VSC. I wrote most of the details on the comment of our checkpoint 2: https://github.com/qiskit-advocate/qamp-fall-22/issues/26#issuecomment-1302034632

**Comment by HuangJunye** [comment]
## Retro

### What went well


### What didn't go well


### What to improve?

**Reply by HuangJunye** [reply]
### What went well
- hacking sessions between Dayeong and Unathi seem to be very productive.
- Great discovery of [pico-pong-online](https://github.com/JRJurman/pico-pong-online) and [pico-socket](https://github.com/JRJurman/pico-socket).

### What didn't go well
- Siddharth is a bit left out from the rest of the team due to busy work schedule.

### What to improve?

**Reply by Unathi-Skosana** [reply]
What went well?
- Dayeong and I had two very fruitful hackathons in the preceding two weeks

What didn't go well?
- No fully functional MVP yet (getting there)

What to improve?
N/A

**Reply by tula3and** [reply]
### what went well

- Found good references through two hackathons!

### what didn't go well

- Didn't make a progress to make a 2-player mode design in PICO-8

### what to improve

- Invest more time to modify QPong PICO-8 based on the references

**Comment by HuangJunye** [comment]
## Planning

### Dayeong

### Unathi

### Siddharth

### Junye

**Reply by HuangJunye** [reply]
- 2h hacking session next Thursday to make MVP of QPong online version
- Deploy server on a cloud service
